### ISSUE TITLE
Change binding or binding settings?

### ISSUE THREAD

**Issue by Kyena** [arnecls/raidplaner#143]
While setting the raidplaner up I had to choose a forum section for the phpbb integration. I used an existing one but made a specific event forum after and would like to change the settings. 

Is it possible to have more than one binding? We use a phpbb forum but also a wordpress site and I would like the option to integrate the raids into a dedicated event section on the website :)

Also, is it possible to create events where you don't have to specify a size (ie open guild events etc).

Thanks in advance!

**Comment by arnecls** [comment]
As you already found out - it is possible to run multiple bindings at once (e.g. wordpress and phpbb).
The only "problem" is that users from both systems are allowed to log in and there might be conflicts.
In that case the binding that is loaded first wins (alphabetical order).

Integration is currently possible by using an iframe. You can also try to do it "inline" but this poses several problems that are not very easy to solve, so I would strongly recommend not to try this.

You can create a raid with attendance mit "just list" which will have a limit (the intended raidsize) but it is ignored. Have a look at the wiki for more information about attendance modes:
https://github.com/arnecls/raidplaner/wiki/Attendance-modes

**Comment by Kyena** [comment]
Hey there :)

Yeah, as a matter of fact I checked out the Wiki after I posted (not so familiar with Github's layout but I am finding my way around now. Thanks for getting back to me!

The only thing I feel is somewhat missing is the possibility to pull a roster overview - however, since I don't add levels etc any more it's not exactly what I was looking for. It's really hard to find a raid planner that has everything I would like. I really appreciate all the work you put into this one, though. Especially with the FFXIV integration :)

**Comment by arnecls** [comment]
There is a kind of roster overview but only for those players who actually raid: The statistics tab in the settings section. You can also query this list by using the API if you want to create your own statistics.

I removed (or better ignored as it was never implemented) the level setting is - from my perspective - it did not make sense anymore to have users in the roster that cannot raid anyway. I've been in about 6 (wow) guilds and it was always like that - before the player reached max level raids were more or less ignored. Once he or she reached the level he or she registered, entered the max level in the raidplaner and went on. Basically no one registered before reaching max level which made the setting totally useless.
When I created the raidplaner I "removed" all fields from existing other solutions that were of no interest to make the whole registration process a lot easier. I could have allowed import from the WoW armory, too, but that would only have worked for WoW and there was (still is?) no official armory API. I.e. you crawled the webpage -> every time blizzard changes the layout of the page, the "API" breaks.

TL;DR - the level field is not in the list as it does not make sense to me (and actually you are the first one the would like to have it). However - if it DOES make sense in some way it is possible to integrate such a field, maybe as an option. But I would really like to hear your usecase.
